cmd/frontend/graphqlbackend/site_admin.go

```diff
@@ -44,13 +44,37 @@ func (r *schemaResolver) RecoverUsers(ctx context.Context, args *RecoverUsersReq
 		ids[index] = id
 	}
 
-	users, err := r.db.Users().RecoverUsersList(ctx, ids)
+	existing, err := r.db.Users().List(ctx, &database.UsersListOptions{
+		UserIDs: ids,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	existingIDs := map[int32]struct{}{}
```

Comment:
nit: I would re-name this like pre-existingIds  or something


```suggestion
	preExistingIDs := map[int32]struct{}{}
```